src/components/expanded-state/asset/ChartExpandedState.js

```diff
@@ -15,8 +15,8 @@ import L2Disclaimer from '../../L2Disclaimer';
 import { ButtonPressAnimation } from '../../animations';
 import { CoinDividerHeight } from '../../coin-divider';
 import CoinDividerOpenButton from '../../coin-divider/CoinDividerOpenButton';
-import EdgeFade from '../../discover-sheet/EdgeFade';
-import UniswapPools from '../../discover-sheet/UniswapPoolsSection';
+import EdgeFade from '../../discover/EdgeFade';
+import UniswapPools from '../../discover/UniswapPoolsSection';
```

Comment:
nit: same here

Reply:
I'll leave UniswapPools in discover section I think, the other one is super generic so I'll move it